examples/LEVDisplay/LEVDisplay.ino

```diff
@@ -0,0 +1,206 @@
+/**********************************************
```

Comment:
There was no merge from the LEV branch so I am not sure how this got in here.

Reply:
Pooh. Seems I need to work at my understanding for github...